app/components/home/HeroCode.js

```diff
@@ -17,14 +17,14 @@ const NewProjectPage: BlitzPage = () => {
 
   return (
     <div>
-      <h1>Create New Project</h1>
+      <h1>Nieuw project aanmaken</h1>
```

Comment:
Hierover was ik ook aan het twijfelen. Woorden in titels met hoofdletter of niet?

Reply:
Hoofdletters in titels is iets typisch Engels. In het Nederlands wordt dat normaal niet gedaan. Dus voor mij liever op deze manier :)

Reply:
Ja, inderdaad dacht ik ook maar was niet zeker.